packages/build/README.md

```diff
@@ -12,7 +12,7 @@ This module contains a set of common scripts and default configurations to build
 
 These scripts first try to locate the CLI from target project dependencies and fall back to bundled ones in `@loopback/build`.
 
-# Usage
+## Usage
```

Comment:
FWIW, according to our [README guidelines](http://loopback.io/doc/en/contrib/README-guidelines.html#basic-use), the correct name for this section is `Basic use`.